lib/modules/accountSwitcher.js

```diff
@@ -106,10 +120,21 @@ module.options = {
 	},
 };
 
-let userLink, $downArrow, $downArrowOverlay;
+let userLink, $downArrow, $downArrowOverlay, isAutoSwitched;
 
 export let $accountMenu;
 
+module.beforeLoad = () => {
+	if (module.options.autoSwitchOnPost.value || module.options.autoSwitchOnComment.value) {
+		isAutoSwitched = sessionStorage.getItem('RESAccountSwitcherAutoSwitched');
```

Comment:
`showNotification` may as well be called directly from here, so that it won't be necessary to add the `autoSwitcher` watcher and thus avoids the `isAutoSwitched` conditionals.